master/reference/calicoctl/resources/policy.md

```diff
@@ -44,17 +44,17 @@ spec:
 | name | The name of the policy. |         | string |
 | annotations | Opaque key/value information to be used by clients. | | map |
 
```

Comment:
The "Accepted Values" column here is entirely devoid of any content. I would suggest deleting the column from this table, since it contains nothing. Alternatively, perhaps you could change the title to "Sample values" and provide some examples, that can really be helpful in a table of this kind.

I would also suggest instead of "Schema" perhaps "Datatype"? That is what it seems to represent.